optaplanner-core/src/main/java/org/optaplanner/core/api/score/Score.java

```diff
@@ -126,6 +126,21 @@
      */
     Score_ negate();
 
+    /**
+     * Returns a Score, all levels of which are zero.
+     *
+     * @return never null
+     */
+    Score_ zero();
```

Comment:
I think that this is a useful addition to the public API, and it helps examples avoid `ScoreDefinition`.

Reply:
If it works for BendableScore (haven't got there yet): yes, good idea!